libpirate/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ SET(PIRATE_SOURCES
     "uio.c"
 )
 
-SET(PIRATE_C_FLAGS -Werror -Wall -Wextra -Wpedantic)
+SET(PIRATE_C_FLAGS -Werror -Wall -Wextra -Wpedantic -fPIE)
```

Comment:
Can we use fPIC here instead of fPIE? That's the only outstanding request I have everything else is good to merge.

Reply:
I am good with that